kairon/async_callback/processor.py

```
@@ -144,15 +144,22 @@ async def process_async_callback_request(token: str,
-                    copied_func = functools.partial(CallbackProcessor.async_callback, rsp, entry, callback, callback_source, bot, entry.get("sender_id"), entry.get("channel"), request_data)
-                    await copied_func()
+                    async def callback_function(rsp: dict):
+                        copied_func = functools.partial(CallbackProcessor.async_callback, rsp, entry, callback,
+                                                        callback_source, bot, entry.get("sender_id"),
+                                                        entry.get("channel"), request_data)
+                        await copied_func()
+
+                    CallbackProcessor.run_pyscript_async(script=callback.get("pyscript_code"),
+                                                         predefined_objects=predefined_objects,
+                                                         callback=callback_function)
+                asyncio.create_task(execute_async_callback())
```

Ruff (0.15.21)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)